src/utils/validations.js

```diff
@@ -194,3 +194,38 @@ export const validateBmcURL = value => {
 
   return validateURL(value);
 };
+
+// Validates that the provided MAC address meets one of following formats supported by the golang ParseMAC function:
+// IEEE 802 MAC-48, EUI-48, EUI-64, or a 20-octet IP over InfiniBand link-layer address
+// https://golang.org/pkg/net/#ParseMAC
+export const isValidMAC = mac => {
```

Comment:
we could also move this function to a separate file

Reply:
At the moment I don't think it's complex enough to warrant its own file, but if it grows in the future I could see doing that.

Reply:
ok we can do it in the future